src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -272,25 +272,27 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
             </TextContainer>
           </CSSTransition>
         </SwitchTransition>
-        <ContextMenu
-          placement="bottom-end"
-          items={
-            publisherMode
-              ? publisherKebabMenuItems
-              : [
-                  {
-                    icon: <SvgGlyphCopy />,
-                    onClick: onCopyVideoURLClick,
-                    title: 'Copy video URL',
-                  },
-                ]
-          }
-          trigger={
-            <KebabMenuButtonIcon onClick={() => null} variant="tertiary" size="small">
-              <SvgGlyphMore />
-            </KebabMenuButtonIcon>
-          }
-        />
+        {!isUploading && (
+          <ContextMenu
+            placement="bottom-end"
+            items={
+              publisherMode
+                ? publisherKebabMenuItems
+                : [
+                    {
+                      icon: <SvgGlyphCopy />,
+                      onClick: onCopyVideoURLClick,
+                      title: 'Copy video URL',
+                    },
+                  ]
+            }
+            trigger={
+              <KebabMenuButtonIcon onClick={() => null} variant="tertiary" size="small" disabled>
+                <SvgGlyphMore />
+              </KebabMenuButtonIcon>
```

Comment:
Why have you added the `disabled` prop here? This makes all the buttons visible right away, without a hover.